content/sensu-core/0.29/reference/filters.md

```diff
@@ -297,6 +297,131 @@ tokens"` will be published to the Sensu server log._
 
 ## Filter configuration
 
+### Built-in Filters - occurrences {#built-in-filters-occurrences}
+
+The `occurrences` filter is included in every install of Sensu. This filter can 
+be applied to a handler using the "filters" handler definition attribute.
+
+For example:
+{{< highlight json >}}
+{
+  "handlers": {
+    "email": {
+      "...": "...",
+      "filters": ["occurrences"]
+    }
+  }
+}
+{{< /highlight >}}
+
+
+The `occurrences` filter uses two custom check definition attributes, `occurrences` 
```

Comment:
Let’s drop the adjective “custom” here. They are more or less part of the spec IMO.